src/components/settings/services/EditServiceForm.tsx

```diff
@@ -4,6 +4,7 @@ import { Link } from 'react-router-dom';
 import { defineMessages, injectIntl, WrappedComponentProps } from 'react-intl';
 import normalizeUrl from 'normalize-url';
 import { mdiInformation } from '@mdi/js';
+import { noop } from 'lodash';
```

Comment:
Should we think of using lodash/noop ??

Doing this will load whole lodash, i do agree this might be already done in other places as well

Some comparisons on optimising lodash import 
https://www.blazemeter.com/blog/import-lodash-libraries

Reply:
lodash is already in use all over the codebase

Reply:
we can discuss further tomorrow, this pr can wait :)

Reply:
lets take that up in a separate PR